RPC integration tests failing in `main`

See https://github.com/stellar/soroban-tools/actions/runs/6284326898/job/17065582660

```
--- FAIL: TestCLICargoTest (651.29s)
    --- FAIL: TestCLICargoTest/integration_and_sandbox::contract_data_read (97.64s)
```

--- a/cmd/crates/soroban-test/tests/it/integration_and_sandbox.rs
+++ b/cmd/crates/soroban-test/tests/it/integration_and_sandbox.rs
@@ -128,7 +128,7 @@ fn contract_data_read() {
         .arg("--durability=persistent")
         .assert()
         .success()
-        .stdout("COUNTER,1,4096\n");
+        .stdout(predicates::str::starts_with("COUNTER,1"));
 
     sandbox
         .new_assert_cmd("contract")
@@ -149,7 +149,7 @@ fn contract_data_read() {
         .arg("--durability=persistent")
         .assert()
         .success()
-        .stdout("COUNTER,2,4096\n");
+        .stdout(predicates::str::starts_with("COUNTER,2"));
 }
 
 #[test]

